source/_includes/puppetdb_master.html

```diff
@@ -29,6 +29,7 @@ <h2 id="puppetdb">PuppetDB NEXT</h2>
   <li><strong>Configuration</strong>
     <ul>
       <li><a href="/puppetdb/{{ puppetdb_version }}/configure.html">Configuring PuppetDB</a></li>
+      <li><a href="/puppetdb/{{ puppetdb_version }}/puppetdb_connection.html">Configuring PuppetDB Connection</a></li>
```

Comment:
What do you think about changing the title of the page? I suggest `puppetdb.conf: Configuring a Puppet/PuppetDB Connection`

Like, it's longer and uglier, but anyone can see exactly whether they need to care about that page immediately, and if someone is looking for puppetdb.conf specifically they won't have to even think about it.